Server.MirForms/Account/PlayerInfoForm.cs

```diff
@@ -356,7 +392,10 @@ private void CurrentIPLabel_Click(object sender, EventArgs e)
 
         private void AccountBanButton_Click(object sender, EventArgs e)
         {
+            if (Character == null) return;
+            if (Character.Player == null) return;
```

Comment:
why only allow account to be banned if player is logged in? Check not needed here - only needed if in game message is to be sent